app/src/main/java/mcgill/shredit/LoginActivity.java

```diff
@@ -69,5 +69,41 @@ public void onLoginClick(View view){
             }
         }
     }
+    
+    public boolean signUpUser () {
+        final String username = editTextUsername.getText().toString().trim();
+        String password = editTextPassword.getText().toString().trim();
+
+        if (TextUtils.isEmpty(username) && TextUtils.isEmpty(password)) {
+            Toast.makeText(this, "Please enter username and password",Toast.LENGTH_SHORT).show();
+
+            return false;
+        }
+        else if(TextUtils.isEmpty(username) && !TextUtils.isEmpty(password)){
+            Toast.makeText(this, "Please enter username ",Toast.LENGTH_SHORT).show();
+
+            return false;
+        }
+        else if(TextUtils.isEmpty(password) && !TextUtils.isEmpty(username)){
+            Toast.makeText(this, "Please enter password",Toast.LENGTH_SHORT).show();
 
+            return false;
+        }
+
+        else if(true){// register
```

Comment:
what is this condition for

Reply:
it was originally like this

  else if (registerUser(String username,String password)){

where boolean registerUser() would query database and return true if register is successful.

Reply:
why did you change it to true then?

Reply:
i dont get why you changed it to true either

Reply:
The method would need a database to query

Reply:
dont we have a database?

Reply:
@jiaweini I just saw your update. We can't use the mock you created. Whosoever is working on the database should build the authentication system. The data in our database should be tied to the user

Reply:
I'll change it if someone could tell me where the user info database is and how to query it.  I'm doing this mock database thing only because I can't find a real database to work with.